Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -748,21 +754,21 @@ void function OnHarvesterDamaged(entity harvester, var damageInfo)
 		float oldhealthpercent = ( ( harvester.GetHealth().tofloat() / harvester.GetMaxHealth() ) * 100 )
 		float healthpercent = ( ( newHealth / harvester.GetMaxHealth() ) * 100 )
 
-		if (healthpercent <= 75 && oldhealthpercent > 75) // we don't want the dialogue to keep saying "Harvester is below 75% health" everytime they take additional damage
+		if ( healthpercent <= 75 && oldhealthpercent > 75 ) // we don't want the dialogue to keep saying "Harvester is below 75% health" everytime they take additional damage
 			PlayFactionDialogueToTeam( "fd_baseHealth75", TEAM_MILITIA )
 
-		if (healthpercent <= 50 && oldhealthpercent > 50)
+		if ( healthpercent <= 50 && oldhealthpercent > 50 )
 			PlayFactionDialogueToTeam( "fd_baseHealth50", TEAM_MILITIA )
 
-		if (healthpercent <= 25 && oldhealthpercent > 25)
+		if ( healthpercent <= 25 && oldhealthpercent > 25 )
 			PlayFactionDialogueToTeam( "fd_baseHealth25", TEAM_MILITIA )
 
-		if (healthpercent <= 10)
+		if  (healthpercent <= 10 )
```

Comment:
```suggestion
		if ( healthpercent <= 10 )
```

Reply:
```suggestion
		if( healthpercent <= 10 )
```

Reply:
I don't really think it matters, Respawn aren't consistent with this :/
![image](https://user-images.githubusercontent.com/66967891/180246987-8d9c3081-0da3-4d83-9ebc-20e57e43d4e9.png)
![image](https://user-images.githubusercontent.com/66967891/180247059-1832a9c3-89ed-437f-a28e-aedd8b7f036e.png)